src/modal/__tests__/modal.test.js

```diff
@@ -175,6 +175,7 @@ describe('Modal', () => {
 
   test('role', () => {
     wrapper = mount(
+      // eslint-disable-next-line jsx-a11y/aria-role
```

Comment:
what's the issue in here?

Reply:
The provided `role` isn't a valid ARIA type. Since we are testing that something is getting passed through I disabled the check since it's a valid test.

Reply:
Same here. Is there a way to override/add some rules?